obspy/io/nordic/ellipse.py

```diff
@@ -0,0 +1,329 @@
+"""
+Routines for error ellipses
+ - Calculate an ellipse from covariance matrix
+ - See if a point is inside an ellipse
+ - Calculate the angle subtended by an ellipse (for back-azimuth uncert)
+ - Plot an ellipse
+
+Still missing:
```

Comment:
This could be a TODO: which should be parsed by sphinx when making the docs to highlight what is missing.

Reply:
OK, changed "Still missing" to "TODO", I hope that's what is needed